### PR TITLE
 Marwa-Week1

### PR DESCRIPTION
**Pull request by Marwa-kwara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ In this module you'll learn all about a fundamental part of any software applica
 
 **Before** your first class you start you need to install the necessary software: MySQL. This differs depending on your operating system:
 
+During installation of MySQL v8, in one of the last steps you will configure the password encryption.
+Here is [a screenshot of the step](https://i.stack.imgur.com/nFnWV.jpg).
```
Comment (unmeshvrije) [inline README.md:18]: This is not part of the homework!

Can you make a new PR ? with only relevant files for the homework.

To suggest changes in the README / reading material, please make a PR to HackYourFuture repository and not to the HackYourHomework repository !

**Review by unmeshvrije — CHANGES_REQUESTED** [review]
Please make a new PR with changes in files relevant to the homework only.

File: README.md (hunk 1 above)
Comment (Marwa-kwara) [reply README.md:18]: okay!
I will make a new PR with changes in files relevant to the homework only, BUT should I make it to HackYourFuture repository  not to the HackYourHomework repository?

**Comment by robvk** [comment]
New one is up

**Pull request closed without merging**